### DISCUSSION TITLE
OT a new Optics package for Julia from Microsoft

### DISCUSSION THREAD

**Discussion by dibyendumajumdar** [mjhoptics/ray-optics#38] [General]
https://github.com/microsoft/OpticSim.jl

Interesting choice of language.

**Comment by mjhoptics** [comment]
Interesting. Appears to be non-sequential ray tracing only, currently. That makes image forming analyses like MTF (not to mention calculating things like focal length) a little more challenging. They claim their code is 15x faster than Zemax on a particular example, which seems too good to be true (although they do comment that other examples might not be as favorable).
